### PR TITLE
[TASK] Validate guides.xml files against scheme

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: packages/guides-cli/resources/schema/guides.xsd [hunk 1]
```diff
@@ -2,12 +2,13 @@
 <xsd:schema
     targetNamespace="https://www.phpdoc.org/guides"
     xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+    xmlns="https://www.phpdoc.org/guides"
```
Comment (jaapio) [inline packages/guides-cli/resources/schema/guides.xsd:5]: we should publish the xsd

Comment (linawolf) [reply packages/guides-cli/resources/schema/guides.xsd:5]: First we should make it work?

**Comment by linawolf** [comment]
@jaapio all tests are green now, please review

File: packages/guides-cli/resources/schema/guides.xsd (hunk 1 above)
Comment (wouterj) [reply packages/guides-cli/resources/schema/guides.xsd:5]: I wouldn't publish the XSD on an uri (but instead reference a local file like done in the examples in this PR). You need proper version management when publishing the XSD's, something you get from free when you recommend people to use the XSD installed in their vendor directory.

(XML namespaces being a "uri" has no relation to the web)

**Review by jaapio — APPROVED** [review]

**Pull request merged**